src/coursier/coursier.ts

```diff
@@ -9,7 +9,10 @@ export function getCoursierExecutable(extensionPath: string): Promise<string> {
       return downloadCoursierIfRequired(
         extensionPath,
         process.platform,
-        "v2.0.13"
+        process.arch,
+        // Have to use a commit hash for now since the launchers repository is untagged
+        "56971135cd9b08eaefed13b4d6b7a95ba9cce572",
```

Comment:
**question:** did you reach out to Alex & co. to see if they can tag these commits in the future?

Reply:
I'll reach out in the coursier repository as an issue.

Reply:
https://github.com/coursier/coursier/issues/2998

Reply:
thankss

Reply:
actually, would you mind putting that link in the code comment?

Reply:
Done